server.js

```diff
@@ -1,248 +1,229 @@
 const inquirer = require('inquirer');
 const mysql = require('mysql');
 const cTable = require('console.table');
+let programIsRunning = true;
+
+const actions = {
+  "View all departments": viewAllDepartments,
+  "View all roles": viewAllRoles,
+  "View all employees": viewAllEmployees,
+  "Add a department": addADepartment,
+  "Add a role": addARole,
+  "Add an employee": addAnEmployee,
+  "Update employee roles": updateEmployeeRoles,
+  "Exit program": exitProgram,
+};
```

Comment:
@samrogers15 Since you have a lot of available options, it'd be easier to manage them via object lookup rather than a large, unwieldy switch statement. Another advantage of this is you then only have to write the action names in a single place rather than duplicate it in the `choices` array in your `startApp` function (note how I replaced the array of strings with a call to `Object.keys` on the `actions` object). 

One caveat is that since we we're referencing the functions at the top of the file but the functions aren't defined until later, I changed your function definitions (defined with an `=` sign) to be function declarations (defined with the `function` keyword). This will allow them to "hoist", meaning that regardless of where you define them in the file, they'll be usable without throwing a reference error. [See more at this link](https://developer.mozilla.org/en-US/docs/Glossary/Hoisting#technical_example).